rust/gateway/Cargo.toml

```
@@ -4,7 +4,23 @@ name = "firezone-gateway"
 version = "1.4.18"
 edition = { workspace = true }
 license = { workspace = true }
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+description = "Gateway for the Firezone zero-trust access product."
+
+[package.metadata.deb]
+maintainer = "Firezone Inc. <support@firezone.dev>"
+copyright = "2025, Firezone Inc."
```

Typo?

legal is `Firezone, Inc.`

```suggestion
copyright = "2025 Firezone, Inc."
```